test-poc/framework/src/main/java/org/keycloak/test/framework/server/AbstractKeycloakTestServerSupplier.java

```diff
@@ -22,10 +24,11 @@ public Class<KeycloakIntegrationTest> getAnnotationClass() {
     @Override
     public InstanceWrapper<KeycloakTestServer, KeycloakIntegrationTest> getValue(Registry registry, KeycloakIntegrationTest annotation) {
```

Comment:
One thing to consider is that RemoteKeycloakTestServer shouldn't use a DatabaseSupplier, as it doesn't need one.